en/install.rst

```diff
@@ -66,3 +74,6 @@ Check if Zephir is available from any directory by executing:
 .. code-block:: bash
 
 	$ zephir help
+
```

Comment:
Any particular reason for this additional whitespace, here? A single newline should suffice, I would think.

Reply:
Was a mistake probably